webrtc.html

```diff
@@ -5462,6 +5462,19 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
     connections between participants. One will normally assume that
     the IP address of TURN servers is not sensitive information.</p>
 
+    <p>Mitigating the exposure of IP addresses to the application requires
+    limiting the IP addresses that can be used, which will impact the
+    ability to communicate on the most direct path between endpoints.
+    Browsers are encouraged to provide appropriate controls for deciding
+    which IP addresses are made available to applications, based on the
+    security posture desired by the user.</p>
+
```

Comment:
I believe that this is still largely in line with what we discussed at the IETF meeting.  That said, I think that Justin wanted some more specific guidance on what the controls might do.  I'd be OK with text to this effect being landed, especially since the following note makes it clear that we intend to consider the matter further.